universum/modules/output/html_output.py

```diff
@@ -2,6 +2,8 @@
 import re
 from datetime import datetime
 
+from ansi2html import Ansi2HTMLConverter
```

Comment:
Conditional import has to be done in different way. See perforce module for example.